extensions/cloud/portability-cloud-microsoft/src/main/java/org/dataportabilityproject/cloud/microsoft/cosmos/AzureTableStore.java

```diff
@@ -164,24 +164,24 @@ public void remove(UUID jobId) throws IOException {
   }
 
   @Override
-  public <T extends DataModel> void create(UUID jobId, T model) {
+  public <T extends DataModel> void create(UUID jobId, String key, T model) {
     try {
-      create(jobId, JOB_DATA_TABLE, null, model);
+      create(createRowKey(jobId, key), JOB_DATA_TABLE, null, model);
     } catch (IOException e) {
       throw new MicrosoftStorageException("Error creating job: " + jobId, e);
     }
   }
 
   @Override
-  public <T extends DataModel> void update(UUID jobId, T model) {}
+  public <T extends DataModel> void update(UUID jobId, String key, T model) {}
```

Comment:
Since this is empty, does it make sense to remove this so the default method can be used? (Which I think throws an unimplemented error)